shared/js/background/chrome-events.es6.js

```diff
@@ -60,6 +85,32 @@ chrome.webRequest.onBeforeRequest.addListener(
     ['blocking']
 )
 
+/**
+ * Listen for script loads from trackers and relay the hosts to the frame's content-script.
+ * This will be used for tracker-dependant decicions in that context.
+ */
+chrome.webRequest.onBeforeRequest.addListener(
+    (details) => {
+        if (trackerutils.isTracker(details.url)) {
+            console.log('tracker script', details.url)
+            chrome.tabs.sendMessage(details.tabId, {
```

Comment:
We could move the code into `handleRequest` in `redirect.es6.js`. However, this may get messy as there are several branches in that code, so to send the message for only the case when we do not block or redirect but it is a tracker may made the code more difficult to read.

Reply:
@kdzwinel so the change needed looks to be ok actually: https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/570/commits/6446343dff44899f54365cd2818af84c78711a6f